src/main/java/org/jenkins/tools/test/model/PluginRemoting.java

```diff
@@ -167,24 +167,20 @@ public PomData retrievePomData() throws PluginSourcesUnavailableException {
 
             String parentNode = xpath.evaluate("/project/parent", doc);
             if (StringUtils.isNotBlank(parentNode)) {
-                LOGGER.log(Level.INFO, parentNode);
+                LOGGER.log(Level.FINE, "Parent POM: {0}", parentNode);
                 parent = new MavenCoordinates(
                         getValueOrFail(doc, xpath, "/project/parent/groupId"),
                         getValueOrFail(doc, xpath, "/project/parent/artifactId"),
                         getValueOrFail(doc, xpath, "/project/parent/version"));
             } else {
-                LOGGER.log(Level.WARNING, "No parent POM reference for artifact {0}, " +
-                                "likely a plugin with Incrementals support is used (Jenkins JEP-305). " +
-                                "Will try to ignore it (FTR https://issues.jenkins-ci.org/browse/JENKINS-55169). " +
-                                "hpiRemoteUrl={1}, pomFile={2}",
-                        new Object[] {artifactId, hpiRemoteUrl, pomFile});
+                LOGGER.log(Level.FINE, "No parent POM for {0}", artifactId);
```

Comment:
Also downgrading this to fine: this is a common scenario, so no need to scare the user with a warning.